pkg/pillar/cmd/zedrouter/acl.go

```diff
@@ -1580,13 +1581,25 @@ func createMarkAndAcceptChain(aclArgs types.AppNetworkACLArgs,
 		return errors.New("Invalid chain creation")
 	}
 
+	chainFlush := []string{"-t", "mangle", "--flush", name}
+
 	newChain := []string{"-t", "mangle", "-N", name}
 	log.Functionf("createMarkAndAcceptChain: Creating new chain (%s)", name)
 	err := iptables.IptableCmd(log, newChain...)
 	if err != nil {
-		log.Errorf("createMarkAndAcceptChain: New chain (%s) creation failed: %s",
+		// if chain already exists, we can skip this error
+		if !strings.Contains(err.Error(), "Chain already exists") {
+			log.Errorf("createMarkAndAcceptChain: New chain (%s) creation failed: %s",
+				name, err)
+			return err
+		}
+		log.Warningf("createMarkAndAcceptChain: New chain (%s) creation failed skipped: %s",
```

Comment:
What this warning says is actually not correct at this point in code.
By this point in code we know that the chain already exists. So it should be an info (Functionf)/debug message saying that the required chain already exists and that we will try to flush the existing rules and add new rules.